frontend/src/components/forms/SvgButton.module.scss

```diff
@@ -11,6 +11,17 @@
   cursor: pointer;
 }
 
+.disabled {
+  line-height: 1;
+  .svgButtonIcon {
+    cursor: default;
+
+    path {
+      fill: var(--icon-disabled-color, $COLOR_BLACK40);
+    }
+  }
+}
+
```

Comment:
```ts
  &:not(.disabled) {
    &:hover path {
      fill: var(--icon-hover-color, $COLOR_BLACK60);
    }
    cursor: pointer;
  }
```

It might be better to just not apply these in `.svgButtonIcon` when disabled, then e.g. the loading cursor would still show when hovering the icons.

(then the `.disabled` class should be added to the icon obviously)